src/PrestaShopBundle/Resources/views/Admin/Module/Includes/modal_import.html.twig

```diff
@@ -13,33 +13,33 @@
                             <div class="module-import-start">
                                 <i class="module-import-start-icon material-icons">cloud_upload</i><br/>
                                 <p class=module-import-start-main-text>
-                                    Drop your module archive here or <a href="#" class="module-import-start-select-manual">select file</a>
+                                    {{ 'Drop your module archive here or [1]select file[/1]'|trans({}, 'Admin.Modules')|replace({'[1]' : '<a href="#" class="module-import-start-select-manual">', '[/1]' : '</a>'})|raw }}
                                 </p>
                                 <p class=module-import-start-footer-text>
-                                    Please upload one file at time, .zip or tarball format (.tar, .tar.gz or .tgz).
-                                    Your module will be installed right after that.
+                                    {{ 'Please upload one file at time, .zip or tarball format (.tar, .tar.gz or .tgz).'|trans({}, 'Admin.Modules') }}
```

Comment:
Yep, thanks for spotting this! I'm changing it, there's another commit coming soon :)